src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2611,6 +2611,103 @@ Task Synchronization API can be used for multiple dynamic synchronization patter
 
 A complete example of Producer / Consumer, _ServiceDeployerConsumer_ can be downloaded link:examples/ServiceDeployerConsumer.xml[here].
 
+[[_task_signal_api]]
+=== Signal API
+
+The *Signal API* is a high-level API that enables users to interact with running jobs via signals (we also call them actions).
+That is, it allows a job to expose specific signals (e.g., `soft_stop` to properly finish a job) that can be triggered by users from the *Workflow Execution portal*.
+
+At a high-level view, the Signal API allows jobs and users to exchange signals using a communication channel provided by the <<_task_synchronization_api,synchronization API>>.
+The typical usage scenario of the Signal API is the following:
+
+  . A job declares to be ready to handle one or more signals. The signals are then displayed in the Workflow Execution portal (e.g., display the `soft_stop` signal)
```

Comment:
asciidoc syntax for unordered list is normally "*"

Reply:
missing . at the end of the sentence

Reply:
It is an ordered list, and the dots are added.